Classifiers/classifier-Crowdstrike-Streaming.json

```diff
@@ -43,7 +43,7 @@
 					"complex": null
 				}
 			},
-			"dontMapEventToLabels": false
+			"dontMapEventToLabels": true
```

Comment:
@meirwah why?

Reply:
because CS streaming have a set of labels the integration creates (parsed).
If I make this false, those labels will be overriden by junk labels.

Reply:
got it,
we need to change this behavior so existing labels won't be overridden (I'll open an issue)

Reply:
yes